pkgs/applications/science/math/geogebra/geogebra_6.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, unzip, fetchurl, electron_6, makeWrapper, geogebra }:
+stdenv.mkDerivation rec{
+
+  name = "geogebra-${version}";
+  version = "6-0-598-0";
+
+  src = fetchurl {
+    url = "https://download.geogebra.org/installers/6.0/GeoGebra-Linux64-Portable-${version}.zip";
```

Comment:
Those URLs usually disappear after a while. You may want to add a second source that points to `web.archive.org` (see `pkgs.geogebra` for this).